canonical-schema.json

```diff
@@ -118,6 +119,11 @@
           , "type"       : "string"
           , "pattern"    : "^[a-z]+(-[a-z]+)*$"
           },
+    
+      "join-strings":
+          { "description": "If true, arrays of strings should be joined to a multiline string"
```

Comment:
I was wondering if we should add something like "arrays of strings in the input or expected properties", to make it clear that this property is only aimed at properties in the input and expected properties.

Reply:
Sounds like a good idea.